omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
+
+# Global FIPS override flag.
+if windows? || rhel?
+  override :fips, enabled: true
+end
```

Comment:
Yeah, we're always building OpenSSL FIPS support on those platforms, but you have to enable it for the client.